kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml

```diff
@@ -125,7 +126,9 @@ data:
           set $target http://report-service.{{ .Values.namespace }}.svc.cluster.local:3030;
           rewrite ^/report/(.*) /$1 break;
           proxy_http_version 1.1;
+{{- if and .Values.nginx_private_ingress_ip (ne .Values.csp "oci") }}
           proxy_set_header Host $server_name;
```

Comment:
@ddevadat why this header is not required for OCI?

Reply:
need to check further

Reply:
in OCI private loadbalancers cant have reserved or predefined ip. Azure lb allows this. 
If we try to define the variable private_ingressgateway_ip with some ip, then nginx-private-ingress deployment will fail.